cmd/agola/cmd/projectsecretlist.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2019 Sorint.lab
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	gwapitypes "agola.io/agola/services/gateway/api/types"
+	gwclient "agola.io/agola/services/gateway/client"
+
+	"github.com/spf13/cobra"
+	errors "golang.org/x/xerrors"
+)
+
+var cmdProjectSecretList = &cobra.Command{
+	Use:   "list",
+	Short: "list project secrets",
+	Run: func(cmd *cobra.Command, args []string) {
+		if err := secretList(cmd, "project", args); err != nil {
+			log.Fatalf("err: %v", err)
+		}
+	},
+}
+
+type secretListOptions struct {
+	parentRef string
+}
+
+var secretListOpts secretListOptions
+
+func init() {
+	flags := cmdProjectSecretList.Flags()
+
+	flags.StringVar(&secretListOpts.parentRef, "project", "", "project id or full path")
+
+	if err := cmdProjectSecretList.MarkFlagRequired("project"); err != nil {
+		log.Fatal(err)
+	}
+
+	cmdProjectSecret.AddCommand(cmdProjectSecretList)
+}
+
+func secretList(cmd *cobra.Command, ownertype string, args []string) error {
+	if err := printSecrets(ownertype, fmt.Sprintf("%s secrets", ownertype), false, false); err != nil {
```

Comment:
Make the written ownertype (project or projectgroup) capitalized like done with `All secrets`.